src/_P118_Itho.ino

```diff
@@ -0,0 +1,548 @@
+//			better to change this value in the Itho-lib code and compile it yourself
+//			svollebreggt, 13-2-2021 - Now uses rewirtten library made by arjenhiemstra: https://github.com/arjenhiemstra/IthoEcoFanRFT
+
+// Recommended to disable RF receive logging to minimize code execution within interrupts
+
+// List of commands:
+// 1111 to join ESP8266 with Itho ventilation unit
+// 9999 to leaveESP8266 with Itho ventilation unit
+// 0 to set Itho ventilation unit to standby
+// 1 - set Itho ventilation unit to low speed
+// 2 - set Itho ventilation unit to medium speed
+// 3 - set Itho ventilation unit to high speed
+// 4 - set Itho ventilation unit to full speed
+// 13 - set itho to high speed with hardware timer (10 min)
+// 23 - set itho to high speed with hardware timer (20 min)
+// 33 - set itho to high speed with hardware timer (30 min)
+
+//List of States:
+
+// 1 - Itho ventilation unit to lowest speed
+// 2 - Itho ventilation unit to medium speed
+// 3 - Itho ventilation unit to high speed
+// 4 - Itho ventilation unit to full speed
+// 13 -Itho to high speed with hardware timer (10 min)
+// 23 -Itho to high speed with hardware timer (20 min)
+// 33 -Itho to high speed with hardware timer (30 min)
+
+// Usage for http (not case sensitive):
+// http://ip/control?cmd=STATE,1111
+// http://ip/control?cmd=STATE,1
+// http://ip/control?cmd=STATE,2
+// http://ip/control?cmd=STATE,3
+
+// usage for example mosquito MQTT
+// mosquitto_pub -t /Fan/cmd -m 'state 1111'
+// mosquitto_pub -t /Fan/cmd -m 'state 1'
+// mosquitto_pub -t /Fan/cmd -m 'state 2'
+// mosquitto_pub -t /Fan/cmd -m 'state 3'
+
+
+// This code needs the library made by 'supersjimmie': https://github.com/supersjimmie/IthoEcoFanRFT/tree/master/Master/Itho
+// A CC1101 868Mhz transmitter is needed
+// See https://gathering.tweakers.net/forum/list_messages/1690945 for more information
+// code/idea was inspired by first release of code from 'Thinkpad'
+
+#include <SPI.h>
+#include "IthoCC1101.h"
+#include "IthoPacket.h"
+#include "_Plugin_Helper.h"
+#ifdef USES_P118
+
+//This extra settings struct is needed because the default settingsstruct doesn't support strings
+struct PLUGIN__ExtraSettingsStruct
+{	char ID1[9];
+	char ID2[9];
+	char ID3[9];
+} PLUGIN_118_ExtraSettings;
+
+IthoCC1101 PLUGIN_118_rf;
+
+// extra for interrupt handling
+bool PLUGIN_118_ITHOhasPacket = false;
+int PLUGIN_118_State=1; // after startup it is assumed that the fan is running low
+int PLUGIN_118_OldState=1;
+int PLUGIN_118_Timer=0;
+int PLUGIN_118_LastIDindex = 0;
+int PLUGIN_118_OldLastIDindex = 0;
+int8_t Plugin_118_IRQ_pin=-1;
+bool PLUGIN_118_InitRunned = false;
+bool PLUGIN_118_Log = false;
+
+// volatile for interrupt function
+volatile bool PLUGIN_118_Int = false;
+//volatile unsigned long PLUGIN_118_Int_time = 0;
+
+#define PLUGIN_118
+#define PLUGIN_ID_118         118
+#define PLUGIN_NAME_118       "Itho ventilation remote"
```

Comment:
All other plugins have a category included in their name. This could be `"Remote - Itho ventilation"` or something similar